### PR TITLE
feat(voip/mumble): Add MumbleSetAudioInputIntent

### PR DESCRIPTION
**Pull request by nikez**

### CODE REVIEW

File: code/components/voip-mumble/src/MumbleAudioInput.cpp [hunk 1]
```diff
@@ -637,3 +670,30 @@ void MumbleAudioInput::ThreadStart(MumbleAudioInput* instance)
...
+		case HashString("speech"):
+		default:
+		{
+			g_curInputIntentMode = InputIntentMode::SPEECH;
+			break;
+		}
+		};
+	});
+
+	OnKillNetworkDone.Connect([=]() {
```
Comment (Disquse) [inline code/components/voip-mumble/src/MumbleAudioInput.cpp:696]: rip bracket :(

File: ext/native-decls/MumbleSetAudioInputIntent.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+ns: CFX
+apiset: client
+game: gta5
+---
+## MUMBLE_SET_AUDIO_INPUT_INTENT
+
+```c
+void MUMBLE_SET_AUDIO_INPUT_INTENT(int intentIndex);
```
Comment (blattersturm) [inline ext/native-decls/MumbleSetAudioInputIntent.md:9]: Hash, since it's a hash.

File: code/components/voip-mumble/component.json [hunk 3]
```diff
@@ -12,7 +12,8 @@
 		"vendor:webrtc-audio-processing",
 		"vendor:labsound",
 		"vendor:speexdsp",
-		"scrbind:base"
+		"scrbind:base",
+		"gta:core:five"
```
Comment (blattersturm) [inline code/components/voip-mumble/component.json:16]: Plain `gta:core` or maybe just ICoreGameInit usage instead of OnKillNetworkDone?

**Review by blattersturm — CHANGES_REQUESTED** [review]
Few changes.

**Review by blattersturm — APPROVED** [review]
Sure. Shame on you for adding 2 extra commits and not just squashing it.

**Pull request merged**